app/controllers/participate/login.js

```diff
@@ -24,7 +24,7 @@ export default Ember.Controller.extend({
           this.transitionToRoute('participate.survey.consent');
         })
         .catch((e) => {
-          if (e.status === 404) {
+          if (e.status === 404 || e.status === 401) {
```

Comment:
How do you think this should be re-arranged? I could add another, more generic error message for everything else that isn't handled.